.ci_support/osx_64_.yaml

```diff
@@ -31,7 +31,7 @@ fftw:
 fortran_compiler:
 - gfortran
 fortran_compiler_version:
-- '9'
+- '11'
```

Comment:
I don’t know how to keep the rerender from updating Fortran which breaks osx-64.